data/tribes/initialization/atlanteans/units.lua

```diff
@@ -1981,8 +1983,12 @@ wl.Descriptions():new_tribe {
          helptexts = {
             -- TRANSLATORS: Purpose helptext for an Atlantean production site: Ferry Yard
             purpose = pgettext("building", "Builds ferries."),
-            -- TRANSLATORS: Note helptext for an Atlantean production site: Ferry Yard
-            note = pgettext("building", "Needs water nearby. Be aware ferries carry wares only, no workers."),
+            note = {
+               -- TRANSLATORS: Note helptext for an Atlantean production site: Ferry Yard, part 1
+               pgettext("building", "Needs water nearby. Be aware ferries carry wares only, no workers."),
```

Comment:
Inherited: Why do only Atlanteans have the hint about "no workers"?

Reply:
good catch please feel free to adopt the other tribes Helptexts as well.